src/main/java/Sb2.java

```diff
@@ -11,23 +11,40 @@
  * @author B. Clint Hall
  */
 public class Sb2 {
+    private static final String NO_JSON = "This project contains no "
```

Comment:
Looks like you've found the checkstyle-approved way to declare a static final field. I'll use this for reference.